### PR TITLE
sgt-puzzles: new package at version 20201208

### PR DESCRIPTION
**Pull request by jslick**
## Description of package

Simon Tatham's Portable Puzzle Collection

Some distros package additional .desktop files, but they are custom by the packagers and I'm too lazy rn.

## Installed manifest of package

```
/var/db/kiss/installed/sgt-puzzles/version
/var/db/kiss/installed/sgt-puzzles/sources
/var/db/kiss/installed/sgt-puzzles/manifest
/var/db/kiss/installed/sgt-puzzles/depends
/var/db/kiss/installed/sgt-puzzles/checksums
/var/db/kiss/installed/sgt-puzzles/build
/var/db/kiss/installed/sgt-puzzles/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/sgt-untangle
/usr/bin/sgt-unruly
/usr/bin/sgt-unequal
/usr/bin/sgt-undead
/usr/bin/sgt-twiddle
/usr/bin/sgt-tracks
/usr/bin/sgt-towers
/usr/bin/sgt-tents
/usr/bin/sgt-solo
/usr/bin/sgt-slant
/usr/bin/sgt-sixteen
/usr/bin/sgt-singles
/usr/bin/sgt-signpost
/usr/bin/sgt-samegame
/usr/bin/sgt-rect
/usr/bin/sgt-range
/usr/bin/sgt-pegs
/usr/bin/sgt-pearl
/usr/bin/sgt-pattern
/usr/bin/sgt-palisade
/usr/bin/sgt-netslide
/usr/bin/sgt-net
/usr/bin/sgt-mines
/usr/bin/sgt-map
/usr/bin/sgt-magnets
/usr/bin/sgt-loopy
/usr/bin/sgt-lightup
/usr/bin/sgt-keen
/usr/bin/sgt-inertia
/usr/bin/sgt-guess
/usr/bin/sgt-galaxies
/usr/bin/sgt-flood
/usr/bin/sgt-flip
/usr/bin/sgt-filling
/usr/bin/sgt-fifteen
/usr/bin/sgt-dominosa
/usr/bin/sgt-cube
/usr/bin/sgt-bridges
/usr/bin/sgt-blackbox
/usr/bin/
/usr/
```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

### CODE REVIEW

File: community/sgt-puzzles/build [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --with-gtk=3
+make
+
+make DESTDIR="$1" install
+
+# Prefix binaries to avoid name conflicts with unrelated packages
```
Comment (dilyn-corner) [inline community/sgt-puzzles/build:10]: You can use --program-prefix=sgt during ./configure, no?

File: community/sgt-puzzles/build [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --with-gtk=3
+make
```
Comment (dilyn-corner) [inline community/sgt-puzzles/build:6]: blank line after ./configure ... and make; no blank line between make and make install

**Review by dilyn-corner — CHANGES_REQUESTED** [review]
What a gloriously adorable package.

File: community/sgt-puzzles/build (hunk 1 above)
Comment (jslick) [reply community/sgt-puzzles/build]: omg I didn't know that existed!  I was going off of the arch PKGBUILD; I guess whoever wrote that didn't know about it either haha

Comment (dilyn-corner) [reply community/sgt-puzzles/build]: I'm convinced most people who write PKGBUILDs don't ever check ./configure --help

Comment (jslick) [reply community/sgt-puzzles/build]: well fwiw I do read ./configure --help , just not the whole thing lol

**Pull request merged**